service/dependencyscanning.yaml

```diff
@@ -0,0 +1,15 @@
+dependency_scanning:
+  image: briangweber/docker-node:carbon
```

Comment:
Why do we need node in this image?

... only because we are running npm in the before script. If we actually just ran npm in the scripts that needed npm, this could be a raw docker image.

Reply:
Correct. I had said as much in https://github.com/Cimpress-MCP/gitlab-continuous-integration/pull/22#discussion_r252837441, but that was autoresolved with the filename change.

It matters that we "have node" in that we need to maintain this image with both. I much prefer using "raw" or "public" images because I have more trust in what they are intended for.

But having it in the before scripts means that I either need a node-java image for all my maven projects, or I can't share any of the deploy/bundle tasks because they expect a before_script which ran `npm install`